inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -85,6 +85,10 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*Behavior.*InferRequestIOBBlobSetLayoutTest.*CanSetOutBlobWithDifferentLayouts.*layout=CN.*targetDevice=(AUTO|MULTI).*)",
         R"(.*Behavior.*InferRequestSetBlobByType.*Batched.*)",
         R"(.*Auto_Behavior.*InferRequestIOBBlobTest.*canProcessDeallocatedOutputBlobAfterGetAndSetBlob.*)",
+        // Issue: 53914. Disabled the reduce to pooling transformation
+        R"(.*smoke_LPT.*Reduce.*Transformation.*)",
```

Comment:
Thanks Maksim for the comment! @v-Golubev Hi Vladislav, Since reduce to pooling transformations are disabled from the CPU plugin side, if it's not proper to just disable the corresponding LPT tests, do we have any plans about how to deal with such changes from the LPT tests side? It'll be great if you could add some comments here. Thanks!

Reply:
@xuchen-intel Hi! To my mind Reduce*Transformation should be enabled because these transformations are processed separately from ConvertReduceToPooling.
Probably some of these tests failed after your changes. This may be due to the fact that LPT tests check for precision by the name of some layers. ConvertReduceToPooling used to change the names of some of the layers (was this replacement: `<reduce_name>` -> `<reduce_name>+"/pool"`), which is not happening now.
Could you please correct files that disribe Reduce*Transformation test-cases for the CPU Plugin? You need to specify the name of the reduce layer after all transformations:
- "Output_original" if Reduce*Transformation passed ("U8" precision is specified)
- "Output" if Reduce*Transformation didn't pass ("FP32" precision is specified)

Path to LPT CPU test-cases: openvino\inference-engine\tests\functional\plugin\cpu\shared_tests_instances\low_precision_transformations

Reply:
Fixed. Thanks Vladislav and Maksim!